paasta_tools/contrib/check_manual_oapi_changes.sh

```diff
@@ -1,15 +1,18 @@
 #!/bin/bash
-set -uo pipefail
+set -o pipefail
```

Comment:
you could use `${PAASTA_SYSTEM_CONFIG_DIR:-}` to avoid having to drop this safeguard, but it's not a big deal.

```
       ${parameter:-word}
              Use Default Values.  If parameter is unset or null, the expansion of word is substituted.  Otherwise, the value of parameter is substituted.
```

Reply:
Thanks, I've made this change!